package.json

```diff
@@ -8,7 +8,10 @@
         "lint:fix": "prettier-eslint **/**/*.{js,json} --write",
         "prettier": "prettier **/**/*.{js,json} --write",
         "test": "jest --forceExit",
-        "build": "babel server --out-dir build",
+        "test-order": "npm test -- --runInBand",
```

Comment:
test order ?

Reply:
i implemented this because --runInBand as defined runs all tests serially in the current process, rather than creating a worker pool of child processes that run tests. This can be useful for debugging. it helped my test stop failing if when it didnt need to

Reply:
removed it